nitro-protocol/src/signatures.ts

```diff
@@ -59,6 +62,91 @@ export async function sign(wallet: Wallet, msgHash: string | Uint8Array): Promis
   return utils.splitSignature(await wallet.signMessage(utils.arrayify(msgHash)));
 }
 
+// Towards #761 https://github.com/statechannels/go-nitro/issues/761
+//
+export type ShortenedVariablePart =
```

Comment:
Maybe better naming is possible...
Options were `ShortVariablePart`, `VariablePartCore`, etc.